packages/core/src/llm/LLM.ts

```diff
@@ -205,12 +205,16 @@ export class OpenAI implements LLM {
   }
 
   get metadata() {
+    const contextWindow =
+      ALL_AVAILABLE_OPENAI_MODELS[
+        this.model as keyof typeof ALL_AVAILABLE_OPENAI_MODELS
+      ]?.contextWindow ?? 1024;
```

Comment:
where does 1024 come from?

Reply:
Good question, I looked together AI document and cannot find a correct value

Reply:
if it's used only in together AI, how about overriding metadata there?

Reply:
yes, I will fix then